### PR TITLE
Add @xichengliudui to hack/OWNERS

### PR DESCRIPTION
**Pull request by xichengliudui**
**What this PR does / why we need it**:

@xichengliudui  continues reviewing PRs which are related to hack/*.sh,
then this adds him to a reviewer to get him involved more.

PRs list: https://github.com/kubernetes/kubernetes/pulls/xichengliudui

### CODE REVIEW

**Comment by xichengliudui** [comment]
/cc @fejta 
ake me more convenient and quick approve, merge.

**Comment by xichengliudui** [comment]
/cc @fejta 
ake me more convenient and quick approve, merge.

**Comment by xichengliudui** [comment]
/cc @fejta 
ake me more convenient and quick approve, merge.

**Comment by fejta** [comment]
Thanks for all the great PRs! This has paid down a bunch of technical debt, making our code more consistent.


There are a bunch of challenging projects in this space that need help and love, for example:

Some examples:
* Delete the deprecated hack/e2e.go and get people to use kubetest explicitly
* Eliminate the hack/e2e-internal directory, safely migrating its functionality elsewhere.
* Between kubetest in k8s.io/test-infra, ginkgo-e2e.sh, test-*.sh, jenkins/*.sh there is a lot of confusing interaction between all these scripts. We'd like to simplify this functionality, reduce the number of scripts and convert complicated logic into golang wherever possible.

Any chance you'd be willing to take some of them on? Ideally starting with a design doc that outlines the strategy you want to pursue?

**Comment by SataQiu** [comment]
Sounds great! There is much meaningful work to be done. @fejta 
I'm very interested in the third one:
```
Between kubetest in k8s.io/test-infra, ginkgo-e2e.sh, test-.sh, jenkins/.sh there is a lot of confusing interaction between all these scripts. We'd like to simplify this functionality, reduce the number of scripts and convert complicated logic into golang wherever possible.
```
Can I get involved?

**Comment by xichengliudui** [comment]
I can certainly accept that, these things are the obligation of every k8s member.

**Comment by fejta** [comment]
Here's our weekly meeting: https://github.com/kubernetes/community/tree/master/sig-testing

Historically @krzyzacy, @BenTheElder and @spiffxp have all been interested in simplifying our scripts. I would try and find a way to chat with them to come up with a design to share with the sig-testing group.

**Comment by fejta** [comment]
And please do get involved!

File: hack/OWNERS [hunk 1]
```diff
@@ -35,3 +35,4 @@ approvers:
   - soltysh # for sig-cli related stuff
   - dims # for local-up-cluster related files
   - spiffxp
+  - xichengliudui
```
Comment (BenTheElder) [inline hack/OWNERS]: You should start in reviewers before approvers https://github.com/kubernetes/community/blob/master/community-membership.md#reviewer

**Comment by BenTheElder** [comment]
Thanks for all the PRs, approvers are supposed to reviewing for a bit longer though 😅 https://github.com/kubernetes/community/blob/master/community-membership.md#requirements-2

**Comment by xichengliudui** [comment]
@fejta absolutely, I will continue to contribute in this area.

**Pull request closed without merging**